docs/fundamentals/setup-tooling.md

```diff
@@ -211,6 +214,18 @@ You add .NET Aspire components to your app like any other NuGet package using Vi
 
 For more information on .NET Aspire components, see [.NET Aspire components overview](components-overview.md).
 
+### Add a hosting component
+
+.NET Aspire hosting components are used to configure various hosting scenarios. Hosting components are differentiated from other components in that they are added to the **.AppHost** project. To add a hosting component to your app, follow these steps:
```

Comment:
Suggestion: ...are used to configure various resources and dependencies an app may depend on or consume.